CMakeLists.txt

```diff
@@ -0,0 +1,857 @@
+cmake_minimum_required(VERSION 3.0)
+
+# Based on configure.ac
+
+project(gperftools VERSION 2.6.1 LANGUAGES C CXX)
+
+set(tcmalloc_so_version 8.5.4)
+set(profiler_so_version 4.14.4)
+
+set(default_build_cpu_profiler ON)
+set(default_build_heap_profiler ON)
+set(default_build_heap_checker ON)
+set(default_build_debugalloc ON)
+set(default_build_minimal OFF)
```

Comment:
CMake conventions dictate using upper case for all variable names.

Reply:
Sure